x/payment/client/testutil/integration_test.go

```diff
@@ -141,7 +142,7 @@ func (s *IntegrationTestSuite) TestSubmitWirePayForMessage() {
 						require.NoError(err)
 						msgSize, err := strconv.ParseUint(e.GetAttributes()[1].GetValue(), 10, 64)
 						require.NoError(err)
-						s.Equal(uint64(0), msgSize%256, "Message length should be multiplication of 256")
+						s.Equal(uint64(0), msgSize%consts.ShareSize, "Message length should be multiplication of 256")
```

Comment:
it might be better to simply check for a specific size, but this PR has been open too long, so I think we can merge without change this yet again

Reply:
I agree. Let's address this in a separate PR (can one of you open an issue before merging?). 
But:
```suggestion
						s.Equal(uint64(0), msgSize%consts.ShareSize, "Message length should be multiples of const.ShareSize=%v", consts.ShareSize)
```